client/quantum_serverless/core/provider.py

```diff
@@ -54,6 +55,16 @@ class Cluster:
     ip_address: Optional[str] = None
     resources: Optional[Dict[str, float]] = None
 
+    def job_client(self) -> Optional[JobSubmissionClient]:
+        """Returns job client for given compute resource
+
+        Returns:
+            job client
+        """
+        if self.host is not None:
+            return JobSubmissionClient(f"http://{self.host}:8265")
```

Comment:
for this hardcoded port created new issue https://github.com/Qiskit-Extensions/quantum-serverless/issues/68